### ISSUE TITLE
[Fabric 1.18.2] Incompatible with First Person Model

### ISSUE THREAD

**Issue by Peppernose145** [cobrasrock/Skin-Swapper#36]
# Basic Information
Minecraft version: 1.18.2  
Loader: Fabric loader 0.13.3-1.18.2  
OS (likely irrelevant): Arch Linux  

Mod list is included in the log file accessed via [Pastebin](https://pastebin.com/8L3Byp3Y) or via the attached [latest.log](https://github.com/cobrasrock/Skin-Swapper/files/8458044/latest.log)
# The problem
When the game starts with both Skin Swapper and [First Person Model](https://www.curseforge.com/minecraft/mc-mods/first-person-model) installed, the game gets stuck in a loop of repeatedly reloading resources (Mojang splash screen). When this happens we get the following error every loop:

```
@Redirect conflict. Skipping firstperson.mixins.json:PlayerRenderMixin->@Redirect::setModelPoseRedirect(Lnet/minecraft/class_1007;Lnet/minecraft/class_742;Lnet/minecraft/class_742;FFLnet/minecraft/class_4587;Lnet/minecraft/class_4597;I)V from mod firstperson with priority 1000, already redirected by skinswapper.mixins.json:preview.PlayerEntityRendererMixin->@Redirect::setModelPoseRedirect(Lnet/minecraft/class_1007;Lnet/minecraft/class_742;)V from mod skinswapper with priority 1100
```
## Relevant Logs
[Pastebin Log](https://pastebin.com/8L3Byp3Y)
[latest.log](https://github.com/cobrasrock/Skin-Swapper/files/8458044/latest.log)

# To reproduce

Launch the game with this mod and *First Person Model* installed

# Other notes

The mod works as expected (within the scope of the problem) without the *First Person Model* mod

**Comment by bladehorton** [comment]
Ever found a solution to this problem? I am having the same troubles myself

**Comment by tr7zw** [comment]
Hey, dev of FirstPerson here(since the crash report says it's my mod, due to Skin-Swapper using a higher priority). The crash is because of the redirect here https://github.com/cobrasrock/Skin-Swapper/blob/45ed59980e1222ba9e266ad2a7e7a91c0ff36692/src/main/java/net/cobrasrock/skinswapper/mixin/preview/PlayerEntityRendererMixin.java . There is no reason for Skin-Swapper to redirect the method. Just hook at return/tail and change the settings if required.

**Comment by cobrasrock** [comment]
fixed by #45
